packages/world/src/systems/libraries/MoveLib.sol

```diff
@@ -151,28 +207,42 @@ library MoveLib {
     }
   }
 
-  function _gravityApplies(Vec3 playerCoord) internal view returns (bool) {
-    Vec3 belowCoord = playerCoord - vec3(0, 1, 0);
+  function runGravity(EntityId playerEntityId, Vec3 playerCoord) public {
```

Comment:
Compared to before, now with stacked players, the bottom player will first fall all the way, and then the player above will fall?

Reply:
yes sir, but I think it kind of result in the same end result?

Reply:
well except if it is a long fall, the top player will also get damaged, but I think that's nice actually lol

Reply:
yee thats nice, just checking my understanding of the new code